modules/kubernetes/kubernetes.nu

```diff
@@ -119,19 +120,20 @@ export def kn [ns: string@"nu-complete kube ns"] {
 export def 'kconf import' [name: string, path: string] {
     let k = (kube-config)
     let d = $k.data
-    let i = (cat $path | from yaml)
-    let c = [{
+    # :FIXME: the order of fields loaded via `from yaml` is unstable
+    let i = (cat $path | yq -o=json | from json)
```

Comment:
same comment as above

Reply:
ok, i will delete one of them, but `FIXME` will remain for furture fixes.